response/ta_get_transaction_msg.c

```diff
@@ -3,20 +3,17 @@
 ta_get_transaction_msg_res_t* ta_get_transaction_msg_res_new() {
   ta_get_transaction_msg_res_t* res = (ta_get_transaction_msg_res_t*)malloc(
       sizeof(ta_get_transaction_msg_res_t));
-  if (res) {
-    res->msg = NULL;
+  if (res != NULL) {
```

Comment:
Why not `return res` instead?

Reply:
Right, it should just `return res`. The motivation is all request/response type can all be the same format.